.github/workflows/Dockerfile

```diff
@@ -28,7 +29,9 @@ RUN if [ "${RHEL_VERSION}" = "8" ]; then \
         dnf install -y --enablerepo=crb perl-IPC-Run; \
     fi
 RUN dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-${RHEL_VERSION}-x86_64/pgdg-redhat-repo-latest.noarch.rpm
-RUN dnf -qy module disable postgresql
+RUN if [ "${RHEL_VERSION}" = "8" ] || ["${RHEL_VERSION}" = "9" ]; then \
+        dnf -qy module disable postgresql; \
```

Comment:
This command is not need anymore from RHEL10?

https://github.com/ossc-db/pg_rman/pull/289#discussion_r2184940090